network/digitalocean_test.go

```diff
@@ -52,6 +52,14 @@ func TestParseNameservers(t *testing.T) {
 	}
 }
 
+func mkInvalidMAC() error {
+	if isGo15 {
```

Comment:
Thought you'd like that. Seemed better than putting mkInvalidMAC in two different files. :-P